pwa/src/templates/templateParts/header/HeaderTemplate.tsx

```diff
@@ -22,69 +22,83 @@ interface HeaderTemplateProps {
 export const HeaderTemplate: React.FC<HeaderTemplateProps> = ({ layoutClassName }) => {
   const { t } = useTranslation();
   const [filters, setFilters] = React.useContext(FiltersContext);
+  const {
+    location: { pathname },
+  } = React.useContext(GatsbyContext);
```

Comment:
You've already declared `pathname` at the bottom of this file (I'd opt to keep this top one and remove the bottom one).

Due to already declaring it, this is breaking code. 

Note: whenever making a pull request, please check if the builds on Github have ran successfully, which is a good indicator for breaking code (see screenshot).

![image](https://user-images.githubusercontent.com/66552932/193518483-7877e3b0-1943-4f0c-800b-4ced9fcd041b.png)